test/cppsuite/test_harness/workload/thread_context.h

```diff
@@ -201,7 +201,7 @@ class thread_context {
      * _max_op_count is the current maximum number of operations that can be executed in the current
      * transaction. _max_op_count will always be <= _max_op.
      */
-    int64_t _min_op, _max_op, _max_op_count;
+    int64_t _min_op, _max_op, _max_op_count = 0;
```

Comment:
In wiredtiger code convention, we separate declaration and initialization.

Reply:
Declaration and initialisation have been done simultaneously in many places in this project as we decided it was fine this way. @luke-pearson , should we have official coding guidelines for c++ syntax ?